plugins/woocommerce/src/StoreApi/Routes/V1/Checkout.php

```diff
@@ -86,7 +86,6 @@ public function get_args() {
 				'methods'             => \WP_REST_Server::CREATABLE,
 				'callback'            => [ $this, 'get_response' ],
 				'permission_callback' => '__return_true',
-				'validate_callback'   => [ $this, 'validate_callback' ],
```

Comment:
These validate callbacks run early, before params are sanitised. This was one of the reasons we wanted the cart to be loaded earlier for all store API requests.